src/Serilog.Sinks.BrowserHttp/Sinks/BrowserHttp/BrowserHttpSink.cs

```diff
@@ -51,7 +51,8 @@ class BrowserHttpSink : PeriodicBatchingSink
 			long? eventBodyLimitBytes,
 			LoggingLevelSwitch levelControlSwitch,
 			int queueSizeLimit,
-			HttpMessageHandler messageHandler)
+			HttpMessageHandler messageHandler,
+			IDictionary<string, string> httpHeaders)
```

Comment:
Could we name this `defaultRequestHeaders` or `requestHeaders` to imply how it's used?